drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/utils/GuidedDecisionTableUtils.java

```diff
@@ -309,25 +310,30 @@ private String getTypeFromDataOracle( final String factType,
 
     private String[] getValueList( final ConditionCol52 col ) {
         if ( col.getValueList() != null && !"".equals( col.getValueList() ) ) {
-            return col.getValueList().split( "," );
+            return getSplit(col.getValueList());
         }
         return new String[ 0 ];
     }
 
+
     private String[] getValueList( final ActionSetFieldCol52 col ) {
         if ( col.getValueList() != null && !"".equals( col.getValueList() ) ) {
-            return col.getValueList().split( "," );
+            return getSplit(col.getValueList());
         }
         return new String[ 0 ];
     }
 
     private String[] getValueList( final ActionInsertFactCol52 col ) {
         if ( col.getValueList() != null && !"".equals( col.getValueList() ) ) {
-            return col.getValueList().split( "," );
+            return getSplit(col.getValueList());
         }
         return new String[ 0 ];
     }
 
+    private String[] getSplit(final String valueList) {
+        return ListSplitter.split(valueList);
```

Comment:
Due to changes in `RuleModelDRLPersistenceImpl`, shouldn't we call `ListSplitter.split("\"",workingValue);` here?

Reply:
I used the single ' quote for the outer quotes. This way " can be used inside. Here any " gets ignored and included.

Now I see a test in master says` "a, b b b  ,c"` should become `"a", " b b b  ", "c"`. Not sure if the ' is needed then.

Reply:
Thinking thinking. Need to figure out how master and this branch are compatible.

Reply:
I mean, I understand that implementation in this PR, assumes *'* as spanner of options like *'John, Peter'*, and it works in UI as implemented, but I am not sure if it then affects somehow backend where *"* is used as a spanner of such options. I mean Imagine in UI *'"A", "B"'* will this backend store correctly?